applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c

```diff
@@ -478,6 +478,11 @@ int broadcast_sink_pa_sync_set(struct bt_le_per_adv_sync *pa_sync, uint32_t broa
 		}
 	}
 
+	/* If broadcast_sink was not in an active stream we still need to clean it up */
+	if (broadcast_sink != NULL) {
+		broadcast_sink_cleanup();
+	}
```

Comment:
I assume `broadcast_sink_cleanup` is setting `broadcast_sink = NULL`, right?

Reply:
yes, it calls delete and then sets it to NULL